app/views/transactionDetails.html

```diff
@@ -2,283 +2,266 @@
-                      </div>
+                  <a data-toggle="collapse" href="#res-headers-table">Show Headers <i class="glyphicon glyphicon-chevron-down"></i></a>
+                  <div id="res-headers-table" class="panel panel-default collapse">
+                    <div class="panel-body">
+                      <table class="table-condensed table-borderless">
+                        <tr ng-repeat="(key, property) in transactionDetails.response.headers">
+                          <th>{{key}}: </th>
+                          <td>{{property}}</td>
+                        </tr>
+                      </table>
                     </div>
+                  </div>
 
+                  <div ng-if="transactionDetails.response.body" style="display: inline-block; width: 100%; margin-top: 15px;">
+                    <h4>Response body</h4>
+                    <div class="hljsFormat" hljs source="transactionDetails.response.body"></div>
+                    <button style="margin-top: 10px;" class="btn btn-primary pull-right" ng-click="viewBodyDetails('Response', transactionDetails.response.body, transactionDetails.response.headers);">View Full Response Body</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
 
-                    <div ng-show="transactionDetails.properties" style="display: inline-block; width: 100%">
-                      <label>Properties: </label>
-                      <div class="data-group">
-                        <div class="data-value">
-                          <div class="content-box-large default-labels-box" style="margin-right: 10px;" ng-repeat="(key, property) in transactionDetails.properties">
-                            {{key}}: <strong>{{property}}</strong>
-                          </div>  
-                        </div>
-                      </div>
-                    </div>
-
+          <div class="row">
+            <!-- Routes -->
+            <div ng-show="transactionDetails.routes">
+              <h3>Non-primary routes</h3>
+              <div class="data-group no-more-tables">
+                <div class="data-value">
+                  <!-- Table with all the Routes logs -->
+                  <table class="table table-striped">
+                    <thead>
+                      <tr>
+                        <th>Route</th>
+                        <th>URL</th>
+                        <th>Status</th>
+                        <th>Response Time</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      <tr class="table-list" ng-repeat="route in transactionDetails.routes" ng-click="viewAddReqResDetails(route)">
+                        <td data-title="Route">{{ route.name }}</td>
+                        <td data-title="URL">{{ route.request.method + ' ' + route.request.host + ':' + route.request.port + '/' + route.request.path + '?' + route.request.querystring }}</td>
```

Comment:
I've adjusted it to hide the method, path and query string on tcp as well as the headers. Does this seems ok:

![image](https://cloud.githubusercontent.com/assets/266576/11145632/ed055c3a-8a0f-11e5-9cfb-fd8a2a1e5859.png)